### DISCUSSION TITLE
Can I add MenuGroup in RibbonDropDownGallery dynamicly?

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#31] [Q&A]
All I want is a menu separator line in RibbonDropDownGallery dropdown Commands list menus, I don't need Category header.

In Ribbon Tools, I can add Menu group, and add Button in the menu group.

I wonder can I add MenuGroup in RibbonDropDownGallery dynamicly?

**Comment by harborsiem** [comment]
MenuGroup is only allowed in DropDownGallery.MenuGroups markup element. Not available for ItemsSource.

**Comment by rulai-yinqiang** [comment]
What types are available for ItemsSource?

Can I add RibbonDropDownGallery in ItemsSource?

**Comment by harborsiem** [comment]
I don't have much experience about controls in ItemsSource. Microsoft documentation don't say anything. But I think the only possible controls are RibbonButton, RibbonToggleButton.
